pkg/sdk/logging/model/output/azurestore.go

```diff
@@ -40,6 +40,9 @@ type _metaAzure interface{} //nolint:deadcode,unused
 type AzureStorage struct {
 	// Path prefix of the files on Azure
 	Path string `json:"path,omitempty"`
+	// Azure Cloud to use. e.g. AzurePublicCloud, AzureChinaCloud, AzureGermanCloud, AzureUSGovernmentCloud
```

Comment:
```suggestion
	// Azure Cloud to use. e.g. AzurePublicCloud, AzureChinaCloud, AzureGermanCloud, AzureUSGovernmentCloud, AZURESTACKCLOUD (keep uppercase).
```
Something like this would do the thing I think :)